platforms/tests/windows_test.go

```diff
@@ -0,0 +1,22 @@
+//go:build windows
```

Comment:
This test is skipped unless `go test -tags 'windows'` is passed on the command line.